content/browser/child_process_launcher_helper.cc

```diff
@@ -96,8 +97,22 @@ ChildProcessLauncherHelper::ChildProcessLauncherHelper(
   relaunch_renderer_process_monitor_timeout_.reset(new TimeoutMonitor(
```

Comment:
It needs TimeoutMonitor instance even without the switch?

Reply:
Ok, let me move 
relaunch_renderer_process_monitor_timeout_.reset(new TimeoutMonitor)
into
if (base::CommandLine::ForCurrentProcess()->HasSwitch(
          switches::kTcpLaunchTimeout)) {}
condition.